docs/release.md

```diff
@@ -0,0 +1,225 @@
+# Apache SkyWalking BanyanDB release guide
+
+This documentation guides the release manager to release the SkyWalking BanyanDB in the Apache Way, and also helps people to check the release for vote.
+
+## Prerequisites
+
+1. Close(if finished, or move to next milestone otherwise) all issues in the current milestone from [skywalking-banyandb](https://github.com/apache/skywalking-banyandb/milestones) and [skywalking](https://github.com/apache/skywalking/milestones), create a new milestone if needed.
+2. Update [CHANGES.md](../CHANGES.md).
+
+
+## Add your GPG public key to Apache svn
+
+1. Log in [id.apache.org](https://id.apache.org/) and submit your key fingerprint.
+
+1. Add your GPG public key into [SkyWalking GPG KEYS](https://dist.apache.org/repos/dist/release/skywalking/KEYS) file, **you can do this only if you are a PMC member**.  You can ask a PMC member for help. **DO NOT override the existed `KEYS` file content, only append your key at the end of the file.**
+
+## Build and sign the source code package
+
+```shell
+export VERSION=<the version to release>
+git clone git@github.com:apache/skywalking-banyandb && cd skywalking-banyandb
+git tag -a "v$VERSION" -m "Release Apache SkyWalking BanyanDB $VERSION"
+git push --tags
+make clean && make release
+```
+
+The `skywalking-banyandb-${VERSION}-bin.tgz`, `skywalking-banyandb-${VERSION}-src.tgz`, and their corresponding `asc`, `sha512`. **In total, six files should be automatically generated in the directory.**
+
+## Upload to Apache svn
+
+```shell
+svn co https://dist.apache.org/repos/dist/dev/skywalking/
+mkdir -p skywalking/banyandb/"$VERSION"
+cp skywalking-banyandb/build/release/skywalking-banyandb*.tgz skywalking/banyandb/"$VERSION"
+cp skywalking-banyandb/build/release/skywalking-banyandb*.tgz.asc skywalking/banyandb/"$VERSION"
+cp skywalking-banyandb/build/release/skywalking-banyandb*.tgz.sha512 skywalking/banyandb/"$VERSION"
+
+cd skywalking/banyandb && svn add "$VERSION" && svn commit -m "Draft Apache SkyWalking BanyanDB release $VERSION"
+```
+
+## Make the internal announcement
```

Comment:
This is optional, do you want this? We skip this for years in many repositories.

Reply:
This piece is from other sub-projects. I will remove it